client_react/src/components/gamePage/GamePage.js

```diff
@@ -31,17 +33,28 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 export default function GamePage(
-    { gameFromGlobalState, fetchGame, groupId, goBack, deletePlayer, addPlayers, balanceTeams }) {
-    let game = gameFromGlobalState;
+    { game,
+        fetchGame,
+        groupId,
+        goBack,
+        deletePlayer,
+        addPlayers,
+        balanceTeams,
+        startVoting,
+        sendVotes,
+        getVotes,
+        votes }) {
     const classes = useStyles();
 
     const [addPlayersDialogOpened, setaddPlayersDialogOpened] = useState(false);
+    const [voteDialogOpened, setVoteDialogOpened] = useState(false);
 
     useEffect(() => {
         const fetch = async () => {
             const action = await fetchGame();
-            if (action) {
-                game = action.game;
+            //TODO remove this when BE send votes with game
```

Comment:
BE is already sending votes with game. Please check games/{gameId}/views endpoint please